bpf/bpf_lxc.c

```diff
@@ -1632,7 +1632,7 @@ ipv6_policy(struct __ctx_buff *ctx, struct ipv6hdr *ip6, int ifindex, __u32 src_
 
 skip_policy_enforcement:
 	if (ret == CT_NEW) {
-#if defined(ENABLE_NODEPORT) && defined(ENABLE_IPSEC)
+#if defined(ENABLE_NODEPORT) && (defined(ENABLE_IPSEC) || defined(ENABLE_SRV6))
 		ct_state_new.node_port = ct_has_nodeport_egress_entry6(get_ct_map6(tuple),
```

Comment:
This part is a bit unexpected - you don't need any corresponding support in `nodeport_rev_dnat_ingress_ipv6()` ?

Reply:
Nice catch, this is not needed. We don't support IPv6 VPN (yet). I'll remove this.